schematic/schemas/data_model_relationships.py

```diff
@@ -235,3 +248,24 @@ def retreive_rel_headers_dict(self, edge: bool) -> dict[str, str]:
                 raise ValueError(f"Did not provide a 'edge_rel' for relationship {rel}")
 
         return rel_headers_dict
+
+    def get_allowed_values(self, relationship: str) -> Optional[list[Any]]:
+        """Gets the allowed values for the relationship
+
+        Args:
+            relationship: The name of the relationship
+
+        Raises:
+            ValueError: If the relationship doesn't exist
+            AssertionError: If the allowed values aren't a list
+
+        Returns:
+             A list of allowed values if they exist, otherwise None
+        """
+        if relationship not in self.relationships_dictionary:
+            raise ValueError(f"Relationship: '{relationship}' not in dictionary")
+        allowed_values = self.relationships_dictionary[relationship].get(
+            "allowed_values"
+        )
```

Comment:
Should this be:

```suggestion
        allowed_values = self.relationships_dictionary[relationship].get(
            "allowed_values", None
        )
```
?

Reply:
Isn't `None` the default?